src/amo/routes.js

```diff
@@ -14,6 +15,7 @@ export default (
     <IndexRoute component={Home} />
     <Route path="addon/:slug/" component={DetailPage} />
     <Route path="addon/:slug/review/:reviewId/" component={AddonReview} />
+    <Route path="categories/:addonType/" component={CategoriesPage} />
```

Comment:
Oh, I see there's an extensions home and an extensions category list (likewise for themes).

Does it maybe make sense to reverse this? So we have `/extensions/` and `/extensions/categories/`?